superset-frontend/src/datasource/DatasourceEditor.jsx

```diff
@@ -78,6 +79,10 @@ const EditLockContainer = styled.div`
   }
 `;
 
+const StyledAlert = styled(Alert)`
+  margin-bottom: 20px;
```

Comment:
```suggestion
  margin-bottom: ${({ theme }) => theme.gridUnit * 5}px;
```
... if it's needed. It would be nice if these didn't need case-by-case style overrides. If you suspect this will be a common occurrence, I'd rather add a class name or prop to the Alert component itself.